internal/querynode/impl.go

```diff
@@ -460,8 +460,12 @@ func (node *QueryNode) Search(ctx context.Context, req *queryPb.SearchRequest) (
 		return failRet, nil
 	}
 
-	tr := timerecord.NewTimeRecorder(fmt.Sprintf("start do search, msgID = %d, fromSharedLeader = %t, vChannel = %s, segmentIDs = %v",
-		msgID, req.GetFromShardLeader(), req.GetDmlChannel(), req.GetSegmentIDs()))
+	log.Debug("start do search",
+		zap.Int64("msgID", msgID),
+		zap.Bool("fromShardLeader", req.GetFromShardLeader()),
+		zap.String("vChannel", req.GetDmlChannel()),
+		zap.Int64s("segmentIDs", req.GetSegmentIDs()))
+	tr := timerecord.NewTimeRecorder("")
```

Comment:
Do it make sense to create a TimeRecorder with not label?

Reply:
> Do it make sense to create a TimeRecorder with not label?

The purpose of this change is not to repeatedly print the header lebel every time tr.Elapse() is called